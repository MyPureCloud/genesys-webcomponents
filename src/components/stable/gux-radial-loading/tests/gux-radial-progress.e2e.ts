import { newE2EPage } from '@stencil/core/testing';

describe('gux-radial-loading', () => {
  [
    '<gux-radial-loading></gux-radial-loading>',
    '<gux-radial-loading context="modal"></gux-radial-loading>',
    '<gux-radial-loading context="input"></gux-radial-loading>',
    '<gux-radial-loading context="full-page"></gux-radial-loading>'
  ].forEach((content, index) => {
    it(`should display component as expected (${index + 1})`, async () => {
      const page = await newE2EPage();

      await page.setContent(content);

      const element = await page.find('gux-radial-loading');

      expect(element.innerHTML).toMatchSnapshot();
    });
  });
});
