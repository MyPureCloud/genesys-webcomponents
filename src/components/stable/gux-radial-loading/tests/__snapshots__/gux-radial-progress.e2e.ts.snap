// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-radial-loading should display component as expected (1) 1`] = `"<div class=\\"gux-spinner-container gux-modal\\" aria-busy=\\"true\\"><div class=\\"gux-spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (2) 1`] = `"<div class=\\"gux-spinner-container gux-modal\\" aria-busy=\\"true\\"><div class=\\"gux-spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (3) 1`] = `"<div class=\\"gux-spinner-container gux-input\\" aria-busy=\\"true\\"><div class=\\"gux-spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (4) 1`] = `"<div class=\\"gux-spinner-container gux-full-page\\" aria-busy=\\"true\\"><div class=\\"gux-spin-circle\\"></div></div>"`;
